src/index.html

```diff
@@ -1,14 +1,26 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Stop watch</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="page">
+
+      <div class="watch">
+        <div class="watch__face">
+
+          <div class="watch__holder"></div>
```

Comment:
```suggestion
    <div class="page">
      <div class="watch">
        <div class="watch__face">
          <div class="watch__holder"></div>
```
Don't leave empty lines between parent and child elements. Better add them between siblings one

Reply:
completed! empty lines now are between holder, minute-arrow and sec-arrow elements